src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -370,13 +370,48 @@ public void It_includes_internals_visible_to()
             AssemblyInfo.Get(assemblyPath)["InternalsVisibleToAttribute"].Should().Be("Tests");
         }
 
+        private static string _cachedCurrentTFM = null;
+
+        private static string CurrentTFM
```

Comment:
Nit: `LatestTargetFramework` would be a better name here, TFM is often used this way but technically refers to a different form of the TargetFramework (such as `.NETCoreApp,Version=v5.0`).